docs/architecture/adr-009-CERT-group.md

```diff
@@ -0,0 +1,30 @@
+# ADR 009: Computer Emergency Response (CERT) Group
+
```

Comment:
Is this an `ADR`? I don't think this effects the codebase? also would recommend having one PR be for one ADR

Reply:
It will affect the code base - new code will be required to implement CERT on top of specialization groups (ADR 007)